app/views/test/piv_cac_authentication_test_subject/new.html.erb

```diff
@@ -53,18 +53,6 @@
               <%= label_tag(:'error_certificate.revoked', 'Revoked certificate') %>
             </div>
           </label>
-          <label class="btn-border col-12 margin-bottom-2">
-            <div class="radio">
-              <%= radio_button_tag(:error, 'certificate.timeout') %>
-              <%= label_tag(:'error_certificate.timeout', 'Certificate timeout') %>
```

Comment:
`certificate.timeout` and `certificate.ocsp_error` are no longer returned from PKI